catalog/ui/search-ui/search-endpoint/src/test/groovy/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolverSpec.groovy

```diff
@@ -24,157 +24,162 @@ import ddf.catalog.source.UnsupportedQueryException
 import ddf.catalog.source.solr.SchemaFields
 import ddf.catalog.source.solr.SolrFilterDelegate
 import ddf.catalog.source.solr.SolrFilterDelegateFactory
-import org.apache.solr.client.solrj.SolrClient
 import org.apache.solr.client.solrj.response.QueryResponse
 import org.apache.solr.common.SolrInputDocument
 import org.apache.solr.common.util.NamedList
 import org.apache.solr.common.util.SimpleOrderedMap
+import org.codice.solr.client.solrj.SolrClient
 import org.opengis.filter.sort.SortBy
 import spock.lang.Specification
 
 import static ddf.catalog.source.solr.SchemaFields.METACARD_TYPE_FIELD_NAME
 
 class FilteringDynamicSchemaResolverSpec extends Specification {
 
-    public static final String ID_TXT_FIELD = "$Metacard.ID$SchemaFields.TEXT_SUFFIX"
+  public static final String ID_TXT_FIELD = "$Metacard.ID$SchemaFields.TEXT_SUFFIX"
 
-    public static final String SOURCE_ID_TXT_FIELD = "source-id$SchemaFields.TEXT_SUFFIX"
+  public static final String SOURCE_ID_TXT_FIELD = "source-id$SchemaFields.TEXT_SUFFIX"
 
-    public static final String TITLE_TXT_FIELD = "$Metacard.TITLE$SchemaFields.TEXT_SUFFIX"
+  public static final String TITLE_TXT_FIELD = "$Metacard.TITLE$SchemaFields.TEXT_SUFFIX"
 
-    public static final String METADATA_TXT_FIELD = "$Metacard.METADATA$SchemaFields.TEXT_SUFFIX"
+  public static final String METADATA_TXT_FIELD = "$Metacard.METADATA$SchemaFields.TEXT_SUFFIX"
 
-    public static final String METADATA_TXT_TOKENIZED_FIELD = "$METADATA_TXT_FIELD$SchemaFields.TOKENIZED"
+  public static
+  final String METADATA_TXT_TOKENIZED_FIELD = "$METADATA_TXT_FIELD$SchemaFields.TOKENIZED"
```

Comment:
✏️ Any reason this is on a separate line (same comment for the next two constants)?

Reply:
we do not have formatters in pace for Groovy and the one Google one used inside IntelliJ does that because the line gets too long for its taste.